### ISSUE TITLE
Building ORT Docker Images within GitLab/Kubernetes

### ISSUE THREAD

**Issue by jens-erdmann** [oss-review-toolkit/ort-ci-gitlab#1]
Due to the fact that kubernetes [deprecated dockershim](https://kubernetes.io/blog/2020/12/02/dockershim-faq/) it is not possible to run docker in docker configurations in kubernetes anymore. [GitLab recommends](https://docs.gitlab.com/ee/ci/docker/using_kaniko.html) to use [google/kaniko](https://github.com/GoogleContainerTools/kaniko). Kaniko is not yet able to build buildkit images and therefore not able to build ORT docker images in kubernetes.

TL;DR: For GitLab/Kubernetes alternative container image build tools need to be supported.

**Comment by jens-erdmann** [comment]
I consider this issue as solved as I was able to build an ORT image in a Kubernetes cluster with Kaniko v1.8.1 today.
